com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/resources/AdminResource.java

```diff
@@ -0,0 +1,42 @@
+package com.sap.cloud.lm.sl.cf.web.resources;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import com.sap.cloud.lm.sl.cf.core.util.ApplicationConfiguration;
+import com.sap.cloud.lm.sl.cf.web.security.AuthorizationChecker;
+import com.sap.cloud.lm.sl.cf.web.util.SecurityContextUtil;
+
+@Path("/admin")
+@Component
+@Scope(value = "singleton")
+public class AdminResource {
```

Comment:
Is there any plans to add more resources to this base resource? If no, then we do not need this maybe?